rllib/core/models/configs.py

```diff
@@ -194,81 +222,154 @@ class CNNEncoderConfig(ModelConfig):
 
     Example:
 
-        Configuration:
-        input_dims = [84, 84, 3]
-        filter_specifiers = [
-            [16, [8, 8], 4],
-            [32, [4, 4], 2],
-        ]
-        filter_activation = "relu"
-        output_dims = [256]
-        output_activation = "linear"
-
-        Resulting stack in pseudocode:
-        Conv2D(in_channels=3, out_channels=16, kernel_size=[8, 8], stride=[4, 4])
-        ReLU()
-        Conv2D(in_channels=16, out_channels=32, kernel_size=[4, 4], stride=[2, 2])
-        ReLU()
-        Conv2D(in_channels=32, out_channels=1, kernel_size=[1, 1], stride=[1, 1])
-        Flatten()
-        Linear(121, 256)
+    .. code-block:: python
+        # Configuration:
+        config = CNNEncoderConfig(
```

Comment:
This will need to be changed int:

```
from ray.rllib.core.models.configs import CNNEncoderConfig

# Configuration:
config = CNNEncoderConfig(
    input_dims=[84, 84, 3],  # must be 3D tensor (image: w x h x C)
    cnn_filter_specifiers=[
        [16, [8, 8], 4],
        [32, [4, 4], 2],
    ],
    cnn_activation="relu",
    output_dims=[256],  # must be 1D tensor
    output_activation="linear",
    use_bias=True,
)
model = config.build(framework="torch")
```

Reply:
... or somethings similar that reflects the changes to "cnn_filter_specifiers" and "cnn_activation"

Reply:
great catch, done.